pkg/kubectl/resource_printer.go

```diff
@@ -342,25 +342,41 @@ type HumanReadablePrinter struct {
 }
 
 // NewHumanReadablePrinter creates a HumanReadablePrinter.
-func NewHumanReadablePrinter(noHeaders, withNamespace bool, wide bool, showAll bool, showLabels bool, absoluteTimestamps bool, columnLabels []string) *HumanReadablePrinter {
+func NewHumanReadablePrinter(printOptions *PrintOptions) *HumanReadablePrinter {
+	if printOptions == nil {
+		printOptions = &PrintOptions{}
+	}
 	printer := &HumanReadablePrinter{
 		handlerMap: make(map[reflect.Type]*handlerEntry),
-		Options: PrintOptions{
-			NoHeaders:          noHeaders,
-			WithNamespace:      withNamespace,
-			WithKind:           false,
-			KindName:           "",
-			Wide:               wide,
-			ShowAll:            showAll,
-			ShowLabels:         showLabels,
-			AbsoluteTimestamps: absoluteTimestamps,
-			ColumnLabels:       columnLabels,
-		},
+		options:    *printOptions,
```

Comment:
Looks like the problem is still there. Did you fix your code based on what those `hack/verify-*.sh` told you to do and re-run them to make sure everything passes?